addons/svm/core/src/commands/deploy_subraph.rs

```diff
@@ -130,10 +138,19 @@ impl CommandImplementation for DeployProgram {
         use txtx_addon_kit::{constants::DESCRIPTION, types::commands::return_synchronous_ok};
 
         use crate::{
-            constants::{BLOCK_HEIGHT, PROGRAM_IDL, SUBGRAPH_NAME, SUBGRAPH_REQUEST},
+            constants::{
+                BLOCK_HEIGHT, PROGRAM_IDL, SUBGRAPH_ENDPOINT_URL, SUBGRAPH_NAME, SUBGRAPH_REQUEST,
+            },
             typing::subgraph::SubgraphRequest,
         };
-        let _rpc = values.get_expected_string(RPC_API_URL)?;
+
+        let network_id = values.get_expected_string(NETWORK_ID)?;
+        let (subgraph_url, do_include_token) = match network_id {
+            "mainnet" | "mainnet-beta" => (MAINNET_SUBGRAPH_ENDPOINT, true),
+            "devnet" => (DEVNET_SUBGRAPH_ENDPOINT, true),
+            "localnet" | _ => (values.get_expected_string(RPC_API_URL)?, false),
+        };
```

Comment:
Ok I see, there could be some confusion here.
`localnet` is pointing to the local rpc, and both `devnet` should be pointing to the same, remote URL.
That being said, I think we're missing something in our payloads: it should be including the `network_id`.

Reply:
Do you think we'll have the same endpoint for both `devnet`/`mainnet` and the request will be parsed to determine the network type and route accordingly? Rather than mainnet being `https://api.txtx.run/v1/svm-subgraphs` and devnet being `https://devnet.api.txtx.run/v1/svm-subgraphs`?

Reply:
yep